pandas/core/base.py

```diff
@@ -148,10 +149,18 @@ class SpecificationError(Exception):
 
 class SelectionMixin:
     """
-    mixin implementing the selection & aggregation interface on a group-like
+    Mixin implementing the selection & aggregation interface on a group-like
```

Comment:
when making changes to docstrings, if possible can you make the first line a one-liner.

Reply:
Made as a one-liner.